server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -366,6 +371,11 @@ public Builder clusterUUIDCommitted(boolean clusterUUIDCommitted) {
             return this;
         }
 
+        // TODO: remove once upload PR is merged.
```

Comment:
Same thing

Reply:
Yes, it is waiting on upload PR: https://github.com/opensearch-project/OpenSearch/pull/10404. 

These methods are introduced in upload PR, once it is merged we will remove this TODO from there.